### PR TITLE
doc: Add README.rst to project root

### PR DESCRIPTION
**Pull request by dbkinder**
Jira: ZEP-2055

Signed-off-by: David B. Kinder <david.b.kinder@intel.com>

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+* **Releases**: Source code for Zephyr kernel releases are available at
+  https://zephyrproject.org/downloads. On this page,
+  you'll find release information, and links to download or clone source
+  code from our GitHub repository.  You'll also find links for the Zephyr
+  SDK, a moderated collection of tools and libraries used to develop your
+  applications.
+
+* **Jenkins CI server**: We use a`Jenkins Continuous Integration (CI) server`_
+  to automate pre-verification of Git submissions, and tasks
+  to automate image building and testing processes.
```
Comment (nashif) [inline README.rst]: remove jenkins

File: README.rst [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+* **IRC Chatting**: You can chat online with the Zephyr project developer
+  community and other users in our IRC channel #zephyrproject on the
+  freenode.net IRC server. You can use the http://webchat.freenode.net web
+  client or use a client-side application such as pidgin.
+
+
+.. _supported boards: https://www.zephyrproject.org/doc/boards/boards.html
+.. _Zephyr Introduction: https://www.zephyrproject.org/doc/introduction/introducing_zephyr.html
+.. _Getting Started Guide: https://www.zephyrproject.org/doc/getting_started/getting_started.html
+.. _Jenkins Continuous Integration (CI) Server: https://jenkins.zephyrproject.org/view/zephyr/
```
Comment (nashif) [inline README.rst]: remove

File: README.rst [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+.. _Contribution Guide: https://github.com/zephyrproject-rtos/zephyr/wiki/Contribution-Guide
+.. _Zephyr GitHub wiki: https://github.com/zephyrproject-rtos/zephyr/wikiA
```
Comment (nashif) [inline README.rst]: extra A at the end

**Review by nashif — CHANGES_REQUESTED** [review]
please remove jenkins sections, we do not use jenkins anymore, a few other comments inline

File: README.rst (hunk 1 above)
Comment (dbkinder) [inline README.rst]: done

File: README.rst (hunk 2 above)
Comment (dbkinder) [inline README.rst]: done

File: README.rst (hunk 3 above)
Comment (dbkinder) [inline README.rst]: darn vi, done

**Comment by dbkinder** [comment]
I used my "ostro" method of using ```commit --amend``` and ```push -f``` to make review changes (so there's only one final commit instead of stacked commits so you can see what that would look like in the new process.

**Comment by nashif** [comment]
Can't see anything new yet. The method you are using is the correct method and the same way we did things in gerrit.

**Comment by dbkinder** [comment]
I can see the updated README.rst (with your review changes) if I click on the "files changed" a the top

**Review by nashif — APPROVED** [review]

**Review by nashif — APPROVED** [review]

File: README.rst [hunk 4]
```diff
@@ -0,0 +1,87 @@
+:orphan:
```
Comment (nashif) [inline README.rst]: remove this, it appears on the main repo page now.

**Pull request merged**